### PR TITLE
🌏 Add "tids", or "translation IDs" to all strings in index.html, in preparation for l10n.

### PR DESCRIPTION
**Pull request by michelle**
r? @asolove-stripe

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -31,17 +31,15 @@
...
-        <p class="common-BodyText">Refer to our <a class="common-Link" href="https://stripe.com/docs/elements">quickstart guide</a> and Stripe.js <a class="common-Link" href="https://stripe.com/docs/stripe.js">reference docs</a> to get started using Elements. If you need help after reading this, search our documentation or check out answers to common questions. You can even chat live with other developers in #stripe on freenode.</p>
+        <h1 data-tid="elements_examples.intro.title" class="common-SuperTitle">Stripe Elements examples</h1>
+        <p data-tid="elements_examples.intro.subtitle" class="common-IntroText">Stripe Elements are pre-built rich UI components that help you build your own pixel-perfect checkout flows across desktop and&nbsp;mobile.</p>
         <p class="common-ButtonGroup">
-        <a href="https://stripe.com/elements" class="common-Button common-Button--default">
+        <a data-tid="elements_examples.intro.button_learn_mode" href="https://stripe.com/elements" class="common-Button common-Button--default">
```
Comment (atty-stripe) [inline index.html]: s/button_learn_mode/button_learn_more

File: index.html [hunk 2]
```diff
@@ -31,17 +31,15 @@
     <section class="container-lg">
       <!--Intro-->
       <div class="cell intro">
-        <h1 class="common-SuperTitle">Stripe Elements examples</h1>
-        <p class="common-IntroText">Stripe Elements are pre-built rich UI components that help you build your own pixel-perfect checkout flows across desktop and mobile.</p>
-        <p class="common-BodyText">Refer to our <a class="common-Link" href="https://stripe.com/docs/elements">quickstart guide</a> and Stripe.js <a class="common-Link" href="https://stripe.com/docs/stripe.js">reference docs</a> to get started using Elements. If you need help after reading this, search our documentation or check out answers to common questions. You can even chat live with other developers in #stripe on freenode.</p>
+        <h1 data-tid="elements_examples.intro.title" class="common-SuperTitle">Stripe Elements examples</h1>
+        <p data-tid="elements_examples.intro.subtitle" class="common-IntroText">Stripe Elements are pre-built rich UI components that help you build your own pixel-perfect checkout flows across desktop and&nbsp;mobile.</p>
```
Comment (atty-stripe) [inline index.html]: Should we unify the names of the classes and tids here?

File: index.html [hunk 3]
```diff
@@ -31,17 +31,15 @@
...
           Learn more
         </a>
-        <a href="https://stripe.com/docs/elements" class="common-Button" ="elements_docs">
+        <a data-tid="elements_examples.intro.button_docs" href="https://stripe.com/docs/stripe-js" class="common-Button" ="elements_docs">
           Explore the docs
         </a>
-        </p>
       </div>
 
       <!--Example 1-->
```
Comment (atty-stripe) [inline index.html]: We should add TIDs to the placeholders below in Example 1.

File: index.html [hunk 4]
```diff
@@ -90,10 +88,10 @@ <h3 class="title">Payment successful</h3>
...
-          <span class="no-charge">Your card won't be charged</span>
+          <span data-tid="elements_examples.caption.no_charge" class="no-charge">Your card won't be charged</span>
           <a class="source" href="https://github.com/stripe/elements-examples/#example-1"><svg width="16px" height="10px" viewBox="0 0 16 10" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
               <path d="M1,8 L12,8 C12.5522847,8 13,8.44771525 13,9 C13,9.55228475 12.5522847,10 12,10 L1,10 C0.44771525,10 6.76353751e-17,9.55228475 0,9 C-6.76353751e-17,8.44771525 0.44771525,8 1,8 L1,8 Z M1,4 L8,4 C8.55228475,4 9,4.44771525 9,5 C9,5.55228475 8.55228475,6 8,6 L1,6 C0.44771525,6 6.76353751e-17,5.55228475 0,5 C-6.76353751e-17,4.44771525 0.44771525,4 1,4 L1,4 Z M1,0 L15,0 C15.5522847,-1.01453063e-16 16,0.44771525 16,1 L16,1 C16,1.55228475 15.5522847,2 15,2 L1,2 C0.44771525,2 6.76353751e-17,1.55228475 0,1 L0,1 L0,1 C-6.76353751e-17,0.44771525 0.44771525,1.01453063e-16 1,0 L1,0 Z" fill="#AAB7C4"></path>
-          </svg>View source on GitHub</a>
+            </svg><span data-tid="elements_examples.caption.view_source">View source on GitHub</span></a>
```
Comment (atty-stripe) [inline index.html]: Indentation might be off; we can also pull the closing </a> tag to a new line. Likewise for all the other captions in the page.

File: index.html [hunk 5]
```diff
@@ -226,11 +224,11 @@ <h3 class="title">Payment successful</h3>
             <!--Stripe Apple Pay element inserted here-->
```
Comment (atty-stripe) [inline index.html]: We should update this comment to mention PRAPI as well :P

File: index.html [hunk 6]
```diff
@@ -226,11 +224,11 @@ <h3 class="title">Payment successful</h3>
...
           </div>
           <fieldset>
-            <legend class="card-only">Pay with card</legend>
-            <legend class="payment-request-available">Or enter card details</legend>
+            <legend class="card-only" data-tid="elements_examples.example4.pay_with_card">Pay with card</legend>
+            <legend class="payment-request-available" data-tid="elements_examples.example4.enter_card_manually">Or enter card details</legend>
             <div class="container">
               <div id="example4-card"></div>
-              <input type="submit" value="Donate">
+              <button type="submit" data-tid="elements_examples.example4.donate_button">Donate</button>
```
Comment (atty-stripe) [inline index.html]: It's probably more flexible to not have `example4` in the TIDs here, so we can reuse the strings in future examples. Would rename to `elements_examples.form.pay_with_card` and so on.

EDIT: looks like we use the same strings with the `form` namespace in the next example. We should unify both.

**Comment by atty-stripe** [comment]
Separately: I wonder if it's worth adding prettier to this repo?

File: index.html (hunk 2 above)
Comment (michelle-stripe) [reply index.html]: How do you mean?

File: index.html (hunk 1 above)
Comment (michelle-stripe) [reply index.html]: Good catch!

File: index.html (hunk 4 above)
Comment (michelle-stripe) [reply index.html]: Until we have some sort of html-prettier, I'd prefer to not make too many manual changes. This is worthwhile, though.

File: index.html (hunk 2 above)
Comment (atty-stripe) [reply index.html]: Unifying title/SuperTitle and subtitle/IntroText? What do you think?

Comment (michelle-stripe) [reply index.html]: In general I think tids and classnames should be totally orthogonal concerns -- lmk if you disagree!

**Comment by michelle** [comment]
Thanks for giving this such a close read!

r? @atty-stripe

File: index.html (hunk 2 above)
Comment (atty-stripe) [reply index.html]: That's fair! Agree we should skip for now.

File: index.html [hunk 7]
```diff
@@ -211,24 +217,27 @@ <h3 class="title" data-tid="elements_examples.success.title">Payment successful<
...
+          </a>
         </div>
       </div>
 
       <!--Example 4-->
       <div class="cell example example4">
         <form>
           <div id="example4-paymentRequest">
-            <!--Stripe Apple Pay element inserted here-->
+            <!--Stripe paymentRequest Element inserted here-->
```
Comment (atty-stripe) [inline index.html]: s/paymentRequest/paymentRequestButton/ :P

File: index.html [hunk 8]
```diff
@@ -255,17 +264,20 @@ <h3 class="title" data-tid="elements_examples.success.title">Payment successful<
...
+          </a>
         </div>
       </div>
 
       <!--Example 5-->
       <div class="cell example example5">
         <form>
           <div id="example5-paymentRequest">
-            <!--Stripe payment request API element inserted here-->
+            <!--Stripe paymentRequest Element inserted here-->
```
Comment (atty-stripe) [inline index.html]: Same as above!

**Pull request merged**